chucks-pick3-ws/src/main/java/software/bytepushers/pick3/services/Pick3PredictionServiceImpl.java

```diff
@@ -1,41 +1,36 @@
 package software.bytepushers.pick3.services;
 
-import com.amazonaws.auth.AWSStaticCredentialsProvider;
-import com.amazonaws.auth.BasicAWSCredentials;
-import com.amazonaws.regions.Regions;
 import com.amazonaws.services.lambda.AWSLambda;
-import com.amazonaws.services.lambda.AWSLambdaClientBuilder;
 import com.amazonaws.services.lambda.invoke.LambdaInvokerFactory;
-import org.springframework.beans.factory.annotation.Value;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Service;
 
-@Profile("aws")
 @Service
+@Profile("aws")
 public class Pick3PredictionServiceImpl implements Pick3PredictionService {
 
-    @Value("${chucks_pick3_aws_access_key}")
-    private String awsAccessKey;
```

Comment:
Are we sure we want to remove the AWS access key now.  I think if we remove it will break the deployment.  Please put revert back and put these changes into the feature branch that deals with the encrypting sensitive information in application.properties.